src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -510,7 +507,7 @@ public String getCustodianOrgId() {
 
 			List<Map<String, Object>> existingDataList = cassandraOperation.getRecordsByProperties(
 					Constants.KEYSPACE_SUNBIRD, Constants.TABLE_SYSTEM_SETTINGS, searchRequest, null);
-			if (CollectionUtils.isNotEmpty(existingDataList)) {
+			if (CollectionUtils.isEmpty(existingDataList)) {
```

Comment:
check the condition? is it correct or wrong?

Reply:
checked and changed also